easybuild/easyblocks/n/namd.py

```diff
@@ -112,7 +132,7 @@ def configure_step(self):
                 self.cfg.update('namd_cfg_opts', "--with-fftw")
             self.cfg.update('namd_cfg_opts', "--fftw-prefix %s" % fftw)
 
-        namd_charm_arch = "--charm-arch %s" % '-'.join(self.cfg['charm_arch'].strip().split(' '))
+        namd_charm_arch = "--charm-arch %s" % '-'.join(self.cfg['charm_arch'].strip().split())
```

Comment:
OK, sorry, this is OK. You're still splitting on spaces in `charm_arch` here, but also handling consecutive spaces correctly.